common/models/wordbreakers/src/default/index.ts

```diff
@@ -340,13 +561,29 @@ namespace wordBreakers {
    * Note that
    * @param character a scalar value
    */
-  function property(character: string): WordBreakProperty {
+  function property(character: string, options?: DefaultWordBreakerOptions): WordBreakProperty {
+    // If there is a customized mapping for the character, prioritize that.
+    if(options?.propertyMapping) {
+      let propName = options.propertyMapping(character);
+      if(propName) {
+        return propertyVal(propName, options);
+      }
+    }
+
     // This MUST be a scalar value.
     // TODO: remove dependence on character.codepointAt()?
     let codepoint = character.codePointAt(0) as number;
+
     return searchForProperty(codepoint, 0, WORD_BREAK_PROPERTY.length - 1);
   }
 
+  function propertyVal(propName: string, options?: DefaultWordBreakerOptions) {
+    const matcher = (name: string) => name.toLowerCase() == propName.toLowerCase()
+
+    const customIndex = options?.customProperties?.findIndex(matcher) ?? -1;
```

Comment:
Do we need a polyfill of `findIndex` for older Android API (Chrome 37)?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex